src/components/GamesList/GamesList.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import GamesList from ".";
+import { GameListEntry } from "../../types"
+import { Segment} from "semantic-ui-react";
+import "semantic-ui-css/semantic.min.css";
+
+export default {
+	title: "Games List Story",
+	component: GamesList,
+};
+
+const gamesListMock: GameListEntry[] = [
+    {   gameID: 2,
+        playerNames: [
+            "Pumpkin",
+            "Ol' Yeller",
+            "Ol' Smelly",
+            "Lassie"
+        ]
+    },
+    {   gameID: 5,
+        playerNames: [
+            "Mr. Peanutbutter",
+        ]
+   }
+];
+
+const onJoinMock = () => {
+    alert("onJoin succeeded")
+}
+
+export const GameListStory = () => (
+    <Segment>
```

Comment:
I thought we were going to put this inside a container rather than a segment.

Reply:
Not really important, though.

Reply:
I think, going through the documentation, that container is mostly for text

Reply:
I'm not sure if we're talking about the same thing. This is the element I think we should use: [Semantic UI container](https://semantic-ui.com/elements/container.html). The idea behind it is just to keep the webpage content from sprawling out from edge to edge. It keeps thinks "contained" without adding extra lines or shadows or effects. It's the kind of component that is putting white spaces on the sides of websites like this one!

This is just a storybook story so it's not a big deal. It does look a little funny to have so many nested segments, though so if we aren't going to use a container I might let the component spread out naturally.

Reply:
I discovered the source of the confusion here. See my comment below.